src/components/custom/other-platforms.js

```diff
@@ -233,7 +233,7 @@ export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redir
             <StyledFlexGridContainer content_width="38.4rem" gap="1rem" grid="3" justify="center">
                 {}
                 {excludetoLowerCase !== 'dtrader' && <TraderCard />}
-                {excludetoLowerCase !== 'dbot' && <BotCard />}
+                <ROW>{excludetoLowerCase !== 'dbot' && <BotCard />}</ROW>
```

Comment:
can we `use is_row` instead? 

```suggestion
                {excludetoLowerCase !== 'dbot' && is_row && <BotCard />}
```

Reply:
Ok